Utils/azure-toolkit-ide-libs/azure-toolkit-ide-common-lib/src/main/java/com/microsoft/azure/toolkit/ide/common/portforwarder/AbstractPortForwarder.java

```diff
@@ -20,9 +20,9 @@
 public abstract class AbstractPortForwarder {
     protected ServerSocketChannel server;
 
-    public void startForward(final int localPort) {
+    public void startForward(ServerSocketChannel server) {
```

Comment:
refactor startForward to 
- start local server
- run `server.accept()` in async thread